packages/ui/.stylelintrc.js

```diff
@@ -0,0 +1,31 @@
+module.exports = {
+	extends: [ '../../.stylelintrc' ],
+	plugins: [ 'stylelint-plugin-logical-css' ],
+	rules: {
+		'declaration-property-max-values': {
+			// Prevents shorthand left/right values (unclear for RTL)
```

Comment:
Does this rule _prevent_ the shorthand, or encourage it? I would have expected a shorthand prevention rule to require a minimum number of values, not set a maximum.

Reply:
It's to prevent it! Are we interpreting "shorthand" in the opposite direction?

Shorthand left/right: `margin: 0 2px 0 0`
Longhand left/right: `margin-right: 2px`

I think you're maybe interpreting it in a way where `margin: 0 2px` is shorthand and `margin: 0 2px 0 0` is longhand.

Reply:
> I think you're maybe interpreting it in a way where `margin: 0 2px` is shorthand and `margin: 0 2px 0 0` is longhand.

That is how I was interpreting it, yes.

Reply:
Feel free to suggest a different code comment if that makes things less ambiguous.

Reply:
Maybe something referring to shorthand in context of "property names" instead of "values" ?

e.g.

```suggestion
		'declaration-property-max-values': {
			// Prevents left/right values with shorthand property names (unclear for RTL)
```